samples/charts/mixed-chart.html

```diff
@@ -0,0 +1,122 @@
+<!doctype html>
+<html>
+
+<head>
+		<title>Mixed Chart</title>
+		<script src="../../dist/Chart.bundle.js"></script>
+		<script src="../utils.js"></script>
+		<style>
+			canvas {
+					-moz-user-select: none;
+					-webkit-user-select: none;
+					-ms-user-select: none;
+			}
+			div {
+				display: flex;
+				flex-wrap: wrap;
+			}
+			div div {
+				flex-grow: 1;
+				width: 30%;
+				margin: 0.5em;
+				height: 20em;
+				min-width: 25em;
+			}
+		</style>
+</head>
+
+<body>
+		<button id="randomizeData">Randomize Data</button>
+		<div id="container">
+		</div>
+		<script>
+			var parents = ['scatter', 'line', 'bubble', 'bar', 'horizontalBar', undefined];
+			var childs = [
+				{type:'scatter', color:'#f844'},
+				{type:'scatter', color:'#f844'},
+				{type:'line', color:'#4f84', fill:false},
+				{type:'bubble', color:'#8f44'},
+				{type:'line', color:'#f484'},
+				{type:'bar', color:'#48f4'},
+				{type:'bar', color:'#84f4'},
+				//{type:'horizontalBar', color:'#84f4'} // not supported yet
+			]
+			var size = 8;
+			var charts = [];
+			parents.forEach(function(parent) {
+				var datasets = [];
+				var c = -1;
+				var labels = [];
+				childs.forEach(function(child) {
+					c++;
+					var data = [];
+					for(var i = 0 ; i < size ; i++) {
```

Comment:
should update all the `if` and `for` statements in this PR to include a space before `(` for consistency with the rest of the Chart.js codebase